manage.py

```diff
@@ -1,6 +1,11 @@
 import os
 import sys
 import venv
+import logging
+
+#logging root configuration
+logging.basicConfig(level=logging.DEBUG, # This will log all levels of messages
```

Comment:
Logging should be INFO by default here to prevent a ton of information on the console.